src/test/java/com/uid2/shared/logback/MaskingLokiJsonEncoderTest.java

```diff
@@ -71,8 +71,8 @@ public void testMaskedExceptionsWithS3(Exception ex, String maskedMessage, Strin
 
         assertAll(
                 "testMaskedExceptionsWithS3",
-                () -> assertEquals(log.split("\n")[0].trim(), maskedMessage),
-                () -> assertTrue(log.split("\n")[1].trim().startsWith(stackTrace))
+                () -> assertEquals(log.split("\n")[0].trim(), maskedMessage, String.format("'%s' must equal '%s'", log.split("\n")[0].trim(), maskedMessage)),
```

Comment:
Where does this change come from?

Reply:
This test fails occasionally locally and I can never figure out why as the test just asserts a boolean. By changing the message, we can see why it fails